src/install.rs

```diff
@@ -636,11 +636,17 @@ fn write_console(mountpoint: &Path, platform: Option<&str>, consoles: &[Console]
 
     // set grub commands
     if grub_commands != metal_spec.grub_commands {
-        let path = mountpoint.join("grub2/grub.cfg");
-        let grub_cfg = fs::read_to_string(&path).context("reading grub.cfg")?;
+        // prefer the new grub2/50_coreos.cfg, but fallback to grub2/grub.cfg
+        let mut name = "grub2/50_coreos.cfg";
+        let mut path = mountpoint.join(name);
+        if !path.exists() {
+            name = "grub2/grub.cfg";
+            path = mountpoint.join(name);
+        }
+        let grub_cfg = fs::read_to_string(&path).context(format!("reading {}", name))?;
         let new_grub_cfg = update_grub_cfg_console_settings(&grub_cfg, &grub_commands)
```

Comment:
Very very minor but style is usually to use `.with_context(|| format!(...))` to avoid string allocation even on non-error paths.

Reply:
ok I updated it.. shamefully I know very little about rust, but have been meaning to change that for a long time :(